src/app.module.ts

```diff
@@ -98,15 +104,27 @@ import { ValidationPipe } from '@utils/validation/validation.pipe';
       playground: true,
       fieldResolverEnhancers: ['guards'],
       sortSchema: true,
+      // context: ({ req }) => ({ req }), vyanakiev toDo - review whether / how we inject gql context
     }),
     DataManagementModule,
     BootstrapModule,
-    MsGraphModule,
     WinstonModule.forRootAsync({
       useClass: WinstonConfigService,
     }),
     SearchModule,
     KonfigModule,
+    AadAccountManagementModule.forRootAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: async (configService: ConfigService) => ({
+        clientID: configService.get<AuthConfig>('aad_ropc')?.clientID as string,
+        clientSecret: '',
```

Comment:
presumably user has to fill it in? Looks strange to have an empty string...

Reply:
This needs to be cleaned up overall, together with the rest of the env variables... The clientSecret is optional - it's mandatory only for some authentication flows (e.g. client credentials grant).

Reply:
https://app.zenhub.com/workspaces/cherrytwist-5ecb98b262ebd9f4aec4194c/issues/cherrytwist/server/765